frontend/src/components/mb-educators.vue

```diff
@@ -1,5 +1,5 @@
 <template lang="pug">
-  div.guidebook
```

Comment:
with pug there is no need to include div when it has a class or id
`.guidebook`
is all that is required
same for all the other div.####